langchain/src/chains/tests/llm_chain.int.test.ts

```diff
@@ -147,3 +147,20 @@ test("Test deserialize", async () => {
 
   // chain === chain2?
 });
+
+test("Test passing a runnable to an LLMChain", async () => {
+  const model = new ChatOpenAI({});
+  const runnableModel = model.bind({
+    stop: ["T-O"],
+  });
+
+  const prompt = PromptTemplate.fromTemplate(
+    "You are a bee --I mean a spelling bee.\nQuestion:{input}"
+  );
+
+  const chain = new LLMChain({ llm: runnableModel, prompt });
+
+  const response = await chain.invoke({ input: "How do you spell today?" });
+  console.log({ response });
+  expect(response).not.toContain("T-O-D-A-Y");
```

Comment:
Not to contain?

Reply:
Also wouldn't put an assertion this specific in an int test

Reply:
Stop tokens are `T-O`.
but tbh, this test might be a little flaky. In like 5 runs I did it would always return something like `... T-O-D-A-Y` but if it were to return in another format this will fail...
If you have another idea on how we can check some bind method works I'm all ears.

Reply:
How about just a bound function call?